save_preferences/save_preferences.js

```diff
@@ -1,18 +1,19 @@
-var allRanges = document.querySelectorAll(".range-wrap");
-allRanges.forEach(function (wrap) {
-    var range = wrap.querySelector(".range");
-    var bubble = wrap.querySelector(".bubble");
-    range.addEventListener("input", function () {
+"use strict";
+const allRanges = document.querySelectorAll(".range-wrap");
```

Comment:
What was the purpose of this bubble?

Reply:
I dont know. It was automatically transformed through typescript.